Update qvector defaults (#1206)
ISISNeutronMuon/MDANSE: Doc/pages/H_conv.rst

Q Vector Sampling
~~~~~~~~~~~~~~~~~

A note on crystals
------------------

If you are simulating a crystal (i.e. a highly ordered, anisotropic material), you will
be interested in calculating the observables such as the :math:`S(Q,\omega)` at specific
points in reciprocal space. You will most likely pick a q-vector generator such as
:ref:`qvectors-reference-DispersionLatticeQVectors`, which will generate a single
lattice vector per calculated data point. The concept of convergence in respect to
q-vector sampling will not apply in this situation.

Isotropic systems
-----------------

For isotropic, disordered systems, the calculation results will be averaged over
vectors with similar :math:`|Q|` and output as a function of :math:`|Q|`. Spherical
averaging is most commonly applied, as many systems can be assumed to be fully isotropic.
:ref:`qvectors-reference-SphericalLatticeQVectors` is normally used for calculating
the quasielastic neutron scattering results in isotropic systems.

Generating the vectors on a sphere
is equivalent to assuming that all the possible scattering geometries are possible,
or rotating the simulated system to calculate the scattering results for different

> orientations of the system. This allows to approximate the correct scattering

Reviewer: ```suggestion
orientations of the system. This allows the code to approximate the correct scattering
```

experiment results which would otherwise require a significantly larger simulation.
A physical sample in a real-life experiment typically contains atoms in

> amounts close to :math:`10^23` (the order of magnitude of the Avogadro number), while

Reviewer: ```suggestion
A physical sample in a real-life experiment typically contains in
the order of :math:`10^23` atoms (the Avogadro number), while
```

many MD trajectories contain less than :math:`10^6` atoms. Therefore, an actual experiment will
sample many more configurations of atoms than it is possible to sample in a single MD simulation.
Performing the calculation on multiple Q vectors maximises the amount the information
that can be extracted from an MD trajectory. At the same time, the computational effort

> of the calculation increases with every additional vector sampled. Also, 

Reviewer: Ends abruptly?

Random vector generation
------------------------

For 1D, 2D and 3D averaging (which could be relevant to calculations on layered,
nematic and completely disordered systems, respectively), there exist three

> vector generators which apply the same approach in different number of dimensions:

Reviewer: ```suggestion
vector generators which apply the same approach in different numbers of dimensions:
```

  * :ref:`qvectors-reference-LinearQVectors`,
  * :ref:`qvectors-reference-CircularQVectors`,
  * :ref:`qvectors-reference-SphericalQVectors`.

Each of these will generate vectors randomly within "shells", following a normal distribution
of :math:`|Q|` around the specified mean value for each shell. The spatial distribution
is constrained to a single line for the
linear generator, while for the other two it will be uniform in angles on a circle
and a sphere, respectively.

You will most likely need the *lattice* versions of these generators, which follow
the same approach, but apply additional processing steps afterwards.

Lattice vector generators
-------------------------

Once the random vector distribution has been created, the lattice vector generators

> round the HKL coordinates of all the vectors to the nearest integer values.

Reviewer: ```suggestion
round the **hkl** coordinates of all the vectors to the nearest integer values.
```
? 

I don't know if we have an agreed format for `hkl`, but since they're vectors this would make more sense to me? Not that important though...
